### PR TITLE
configure installation dirs

### PR DESCRIPTION
**Pull request by sjacek**
Small modifications in .pro file and adding .prf file.
Now you can use log4qt just by `CONFIG += log4qt` in your qmake .pro files.

### CODE REVIEW

**Comment by sjacek** [comment]
May I have any details from appveyor?

File: src/log4qt/log4qt.prf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+LIBS += -llog4qt
+INCLUDEPATH += /opt/Qt/5.15.2/gcc_64/include/log4qt
```
Comment (andibacher) [inline src/log4qt/log4qt.prf:2]: this points to a certain qt version on a linux system. do you need this file?

File: src/log4qt/log4qt.pro [hunk 2]
```diff
@@ -23,19 +30,25 @@ DESTDIR = ../../bin
 DEFINES += NOMINMAX QT_DEPRECATED_WARNINGS QT_NO_CAST_FROM_BYTEARRAY QT_USE_QSTRINGBUILDER
 DEFINES += LOG4QT_LIBRARY
 
-target.path = $$INSTALL_PREFIX/lib$$LIB_SUFFIX
+# generate feature file by qmake based on this *.in file.
+QMAKE_SUBSTITUTES += log4qt.prf.in
+OTHER_FILES += log4qt.prf.in
+
+target.path = $$[QT_INSTALL_LIBS]
```
Comment (andibacher) [inline src/log4qt/log4qt.pro:37]: Why do you want to install lib and headers into the Qt installation directory?

File: src/log4qt/log4qt.pro [hunk 3]
```diff
@@ -23,19 +30,25 @@ DESTDIR = ../../bin
...
 INSTALLS = target
 
 header_base.files = $$HEADERS_BASE
-header_base.path = $$INSTALL_PREFIX/include/log4qt
+header_base.path = $$[QT_INSTALL_HEADERS]/log4qt
 INSTALLS += header_base
 header_helpers.files = $$HEADERS_HELPERS
-header_helpers.path = $$INSTALL_PREFIX/include/log4qt/helpers
+header_helpers.path = $$[QT_INSTALL_HEADERS]/log4qt/helpers
```
Comment (andibacher) [inline src/log4qt/log4qt.pro:44]: Why do you want to install  headers into the Qt installation directory?

**Review by andibacher — COMMENTED** [review]
Thanks for you contribution. I have some comments.

**Comment by andibacher** [comment]
As I did not receive any feedback for my comments I close this pull request.

**Pull request closed without merging**